changelogs/fragments/2122-add-ipa-otpconfig-module.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - Add new module: ipa_otpconfig.  (https://github.com/ansible-collections/community.general/pull/2122)
```

Comment:
Please remove this. New plugins and modules must not have changelog fragments (only jinja2 test and filter plugins do).

Reply:
Sorry!  I guess I can't get that right.  When I saw your note that I missed it in my other PR, I went ahead and did that here.  My mistake.